src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { parseHtml } from 'utils/parse-html';
+
+export type DescriptionProps = {
+  title: string;
+  markup: string;
+  url: string;
+  name: string;
+};
+
+type DescriptionListProps = {
+  items: DescriptionProps[];
+  id: string;
+};
+
+export const DescriptionList = ({ items, id }: DescriptionListProps) => (
```

Comment:
`<dl>` is a **definition** list. Should this be `DefinitionList` or was DescriptionList intentional?

Reply:
Hi .. this was intentional!

I considered `DefinitionList` but thinking about it, it felt like `DescriptionLIst` was more appropriate as we're providing a specific visual treatment, which happens to use `<dl>` elements semantically.